actors/builtin/miner/miner_actor.go

```diff
@@ -242,7 +242,7 @@ type SubmitWindowedPoStParams struct {
 	Partitions []PoStPartition
 	// Array of proofs, one per distinct registered proof type present in the sectors being proven.
 	// In the usual case of a single proof type, this array will always have a single element (independent of number of partitions).
-	Proofs []abi.PoStProof
+	Proofs []abi.PoStProof // PARCHECK: arbitrary size, proof type not registered, proofs can have arbitrary sizes.
```

Comment:
These will be checked by the runtime, but I'm not sure what assumptions the runtime makes.

Reply:
TL;DR: is it safe to trust the runtime to verify proofs and handle arbitrarily sized proofs?

Reply:
Probably safe to rely on message size limits, but an explicit bound would still be even safer.